src/index.html

```diff
@@ -24,16 +24,16 @@
 
 <body>
     <section class="row">
-        <h1 class="col-12 fw-bold my-4 mx-5">Leaderboard</h1>
+        <h1 class="col-12 fw-bold my-4 mx-5 text-light">Leaderboard</h1>
         <div class="col-7 left-align">
             <div class="row my-3">
-                <h4 class="col-3 fw-bold">Recent Scores</h4>
+                <h4 class="col-3 fw-bold text-light">Recent Scores</h4>
                 <button class="col-2" id="refresh" type="button">Refresh</button>
             </div>
             <ul class="laderList col-5"></ul>
         </div>
         <div class="col-3">
-            <h4 class="fw-bold">Add your score</h4>
+            <h4 class="fw-bold text-light">Add your score</h4>
             <form>
```

Comment:
- [ ] OPTIONAL
It will be great if you can adjust the `submit` button to always be aligned to the right together with the input fields regardless of the desktop screen size.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/188129437-dfc10412-1ffc-4e94-90c5-b72ab295a91c.png)

**Provided wireframe:**
![image](https://user-images.githubusercontent.com/12745474/188129659-e28672b3-3ef8-49bd-b60d-56fe8801aa12.png)